style.css

```diff
@@ -68,46 +65,333 @@
     line-height: 24px;
     color: #344563;
     margin-bottom: 10px;
-    padding-left: 20px;
   }
 
   .social-media {
     display: flex;
     align-items: center;
-    gap: 10px;
+    gap: 18px;
+    margin-bottom: 15px;
+  }
+
+  .link-text a {
+    text-decoration: none;
   }
 
   .social-media li {
     list-style: none;
-    padding-left: 20px;
   }
 
   .social-media li a {
     text-decoration: none;
     color: #505f79;
   }
 
+  .language-set a img {
+    padding-left: 20px;
+    margin-top: 10px;
+  }
+
   .social-media li a img {
     width: 20px;
   }
 
-  .btn {
+  .link-text {
     font-size: 17px;
     font-weight: 500;
-    text-align: center;
-    padding: 15px;
     width: 200px;
     color: #172b4d;
     margin-bottom: 10px;
     cursor: pointer;
     transition: 0.3;
-    margin-left: 20px;
   }
 
-  .btn:hover {
-    background-color: #5e6c84;
+  /* Works Section */
+
+  .works-section {
+    display: grid;
+    grid-template-rows: repeat(4);
+    padding: 200 24px;
+    gap: 86px;
+  }
+
+  .work-card {
+    display: flex;
+    flex-direction: column;
+    padding: 25px;
+    font-family: 'Poppins', sans-serif;
+  }
```

Comment:
Well done on the design 💯 but this section would look even more like the Figma (cooler 😎 ) if you gave it some round border like it's supposed to have. See below 👇🏽 

## Your design
![Screenshot 2023-01-13 at 08 06 51](https://user-images.githubusercontent.com/45739963/212326995-79928e20-9c59-4af2-80c3-594c6d5be69a.png)

## Figma Design
![Screenshot 2023-01-13 at 08 07 25](https://user-images.githubusercontent.com/45739963/212327066-36936fce-b878-4d88-82d0-0b1cc78615b0.png)